check.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+echo -n "Checking for mdbook-linkcheck ... "
+if ( ! command -v mdbook-linkcheck ); then
```

Comment:
I'm late but why do we need to put this check (and others) in a subshell?

Reply:
Not sure, actually. Is it too much of an issue?

Reply:
I don't like `fork(2)` a new process for the subshell while we can check in the main shell just fine. Save a bit time, and typing :p